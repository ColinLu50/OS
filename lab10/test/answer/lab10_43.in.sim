22777
1346
1366
831
1342
793
