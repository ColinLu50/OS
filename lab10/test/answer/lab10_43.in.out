FCFS
559 -> 240 -> 425 -> 360 -> 295 -> 627 -> 561 -> 203 -> 698 -> 230 -> 268 -> 368 -> 561 -> 190 -> 106 -> 255 -> 228 -> 723 -> 406 -> 796 -> 353 -> 465 -> 319 -> 251 -> 550 -> 424 -> 12 -> 81 -> 117 -> 109 -> 799 -> 280 -> 673 -> 777 -> 358 -> 15 -> 436 -> 131 -> 260 -> 51 -> 278 -> 597 -> 648 -> 118 -> 198 -> 217 -> 801 -> 475 -> 666 -> 123 -> 456 -> 219 -> 412 -> 153 -> 42 -> 642 -> 618 -> 474 -> 781 -> 625 -> 357 -> 458 -> 597 -> 221 -> 506 -> 459 -> 48 -> 167 -> 109 -> 329 -> 136 -> 474 -> 657 -> 658 -> 304 -> 164 -> 499 -> 653 -> 762 -> 88 -> 65 -> 401 -> 271 -> 807 -> 234 -> 375 -> 93 -> 293 -> 516 -> 28 -> 366
22777
SSTF
559 -> 561 -> 561 -> 550 -> 516 -> 506 -> 499 -> 475 -> 474 -> 474 -> 465 -> 459 -> 458 -> 456 -> 436 -> 425 -> 424 -> 412 -> 406 -> 401 -> 375 -> 368 -> 366 -> 360 -> 358 -> 357 -> 353 -> 329 -> 319 -> 304 -> 295 -> 293 -> 280 -> 278 -> 271 -> 268 -> 260 -> 255 -> 251 -> 240 -> 234 -> 230 -> 228 -> 221 -> 219 -> 217 -> 203 -> 198 -> 190 -> 167 -> 164 -> 153 -> 136 -> 131 -> 123 -> 118 -> 117 -> 109 -> 109 -> 106 -> 93 -> 88 -> 81 -> 65 -> 51 -> 48 -> 42 -> 28 -> 15 -> 12 -> 597 -> 597 -> 618 -> 625 -> 627 -> 642 -> 648 -> 653 -> 657 -> 658 -> 666 -> 673 -> 698 -> 723 -> 762 -> 777 -> 781 -> 796 -> 799 -> 801 -> 807
1346
SCAN
559 -> 550 -> 516 -> 506 -> 499 -> 475 -> 474 -> 465 -> 459 -> 458 -> 456 -> 436 -> 425 -> 424 -> 412 -> 406 -> 401 -> 375 -> 368 -> 366 -> 360 -> 358 -> 357 -> 353 -> 329 -> 319 -> 304 -> 295 -> 293 -> 280 -> 278 -> 271 -> 268 -> 260 -> 255 -> 251 -> 240 -> 234 -> 230 -> 228 -> 221 -> 219 -> 217 -> 203 -> 198 -> 190 -> 167 -> 164 -> 153 -> 136 -> 131 -> 123 -> 118 -> 117 -> 109 -> 106 -> 93 -> 88 -> 81 -> 65 -> 51 -> 48 -> 42 -> 28 -> 15 -> 12 -> 0 -> 561 -> 597 -> 618 -> 625 -> 627 -> 642 -> 648 -> 653 -> 657 -> 658 -> 666 -> 673 -> 698 -> 723 -> 762 -> 777 -> 781 -> 796 -> 799 -> 801 -> 807
1366
C-SCAN
559 -> 550 -> 516 -> 506 -> 499 -> 475 -> 474 -> 465 -> 459 -> 458 -> 456 -> 436 -> 425 -> 424 -> 412 -> 406 -> 401 -> 375 -> 368 -> 366 -> 360 -> 358 -> 357 -> 353 -> 329 -> 319 -> 304 -> 295 -> 293 -> 280 -> 278 -> 271 -> 268 -> 260 -> 255 -> 251 -> 240 -> 234 -> 230 -> 228 -> 221 -> 219 -> 217 -> 203 -> 198 -> 190 -> 167 -> 164 -> 153 -> 136 -> 131 -> 123 -> 118 -> 117 -> 109 -> 106 -> 93 -> 88 -> 81 -> 65 -> 51 -> 48 -> 42 -> 28 -> 15 -> 12 -> 0 -> 833 -> 807 -> 801 -> 799 -> 796 -> 781 -> 777 -> 762 -> 723 -> 698 -> 673 -> 666 -> 658 -> 657 -> 653 -> 648 -> 642 -> 627 -> 625 -> 618 -> 597 -> 561
831
LOOK
559 -> 550 -> 516 -> 506 -> 499 -> 475 -> 474 -> 465 -> 459 -> 458 -> 456 -> 436 -> 425 -> 424 -> 412 -> 406 -> 401 -> 375 -> 368 -> 366 -> 360 -> 358 -> 357 -> 353 -> 329 -> 319 -> 304 -> 295 -> 293 -> 280 -> 278 -> 271 -> 268 -> 260 -> 255 -> 251 -> 240 -> 234 -> 230 -> 228 -> 221 -> 219 -> 217 -> 203 -> 198 -> 190 -> 167 -> 164 -> 153 -> 136 -> 131 -> 123 -> 118 -> 117 -> 109 -> 106 -> 93 -> 88 -> 81 -> 65 -> 51 -> 48 -> 42 -> 28 -> 15 -> 12 -> 561 -> 597 -> 618 -> 625 -> 627 -> 642 -> 648 -> 653 -> 657 -> 658 -> 666 -> 673 -> 698 -> 723 -> 762 -> 777 -> 781 -> 796 -> 799 -> 801 -> 807
1342
C-LOOK
559 -> 550 -> 516 -> 506 -> 499 -> 475 -> 474 -> 465 -> 459 -> 458 -> 456 -> 436 -> 425 -> 424 -> 412 -> 406 -> 401 -> 375 -> 368 -> 366 -> 360 -> 358 -> 357 -> 353 -> 329 -> 319 -> 304 -> 295 -> 293 -> 280 -> 278 -> 271 -> 268 -> 260 -> 255 -> 251 -> 240 -> 234 -> 230 -> 228 -> 221 -> 219 -> 217 -> 203 -> 198 -> 190 -> 167 -> 164 -> 153 -> 136 -> 131 -> 123 -> 118 -> 117 -> 109 -> 106 -> 93 -> 88 -> 81 -> 65 -> 51 -> 48 -> 42 -> 28 -> 15 -> 12 -> 807 -> 801 -> 799 -> 796 -> 781 -> 777 -> 762 -> 723 -> 698 -> 673 -> 666 -> 658 -> 657 -> 653 -> 648 -> 642 -> 627 -> 625 -> 618 -> 597 -> 561
793
